src/Generator/PropertyGenerator.php

```diff
@@ -4,13 +4,17 @@
 
 use Laminas\Code\Reflection\PropertyReflection;
 
+use function array_reduce;
+use function method_exists;
+use function PHPUnit\Framework\assertIsBool;
 use function sprintf;
 use function str_replace;
 use function strtolower;
 
 class PropertyGenerator extends AbstractMemberGenerator
 {
     public const FLAG_CONSTANT = 0x08;
+    public const FLAG_READONLY = 0x80;
```

Comment:
I wonder if we should set this to `private` :thinking:

Reply:
Totally, changed!

Reply:
Reverted since we refer to it from tests.